src/mango/src/mango_cursor.erl

```diff
@@ -182,6 +182,7 @@ maybe_add_warning_int(ok, _, UserAcc) ->
    UserAcc;
 
 maybe_add_warning_int(Warning, UserFun, UserAcc) ->
+    couch_stats:increment_counter([mango, unindexed_queries]),
```

Comment:
Don't you need a `case` here to check if `UseIndexInvalidWarning` from above is actually `ok` (sourced from `NoIndexWarning`, in which case there wouldn't be a warning?

Reply:
the function clause in line 181 catches the `ok` case if I read things correctly. This function clause just unconditionally prints a warning, so I assumed I can just increment here without any additional logic

Reply:
definitely not the most obvs code